src/components/MoneyRequestConfirmationList.js

```diff
@@ -767,10 +767,18 @@ function MoneyRequestConfirmationList(props) {
                             numberOfLinesTitle={2}
                             onPress={() => {
                                 if (props.isEditingSplitBill) {
-                                    Navigation.navigate(ROUTES.EDIT_SPLIT_BILL.getRoute(props.reportID, props.reportActionID, CONST.EDIT_REQUEST_FIELD.CATEGORY));
+                                    Navigation.navigate(
+                                        ROUTES.MONEY_REQUEST_STEP_CATEGORY.getRoute(
+                                            CONST.IOU.ACTION.EDIT,
+                                            props.iouType,
+                                            props.transaction.transactionID,
+                                            props.reportID,
+                                            Navigation.getActiveRouteWithoutParams(),
+                                        ),
+                                    );
                                     return;
                                 }
-                                Navigation.navigate(ROUTES.MONEY_REQUEST_CATEGORY.getRoute(props.iouType, props.reportID));
+                                Navigation.navigate(ROUTES.MONEY_REQUEST_TAG.getRoute(props.iouType, props.reportID));
```

Comment:
What's up with this change?

Reply:
Thanks for your catch. @dukenv0307 It seems you update this in this [commit](https://github.com/Expensify/App/pull/35458/commits/6438c074738652dc38b7c4558f013ace0acab14c)